wot-ig-2023-draft.html

```diff
@@ -383,13 +383,12 @@ <h3 id="marketing">Marketing and Outreach</h3>
         <p>To reach out and collaborate with the community as mentioned
         in the scope, the Marketing Task Force works on promoting the
         W3C Web of Things in different ways such as Web presence, Social
-        Media presence, and more. 
-<!-- In the last charter period, we have
-        worked on establishing our website, created an explainer video
-        for the WoT, increased our social media presence, and worked on
-        documenting our work in a more approachable way.
--->
-</p>
+        Media presence, and more. </p>
+	      
+	<p>      
+ 	The Marketing Task Force will also use the opportunity to do outreach with community groups such as with W3C Web of Things Community Group. 
```

Comment:
```suggestion
 	The Marketing Task Force will also use the opportunity to do outreach with community groups such as with the W3C Web of Things Community Group. 
```

Not entirely sure but for me a "the" is missing